### PR TITLE
Update readme.md and list implemented tests.

### PR DESCRIPTION
**Pull request by andresusanopinto**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,33 +1,54 @@
...
+* [Multiple peerconnections](https://test.webrtc.org/manual/multiple-peerconnections/)
+* [Multiple video devices](https://test.webrtc.org/manual/multiple-video-devices/)
+* [Multiple video streams](https://test.webrtc.org/manual/multiple-video/)
+* [Peer2peer](https://test.webrtc.org/manual/peer2peer/)
+* [Peer2peer iframe](https://test.webrtc.org/manual/peer2peer-iframe/)
+* [Single audio stream](https://test.webrtc.org/manual/single-audio/)
+* [Single video stream](https://test.webrtc.org/manual/single-video/)
+
+
+## Contributting ##
```
Comment (KaptenJansson) [inline README.md]: Should be Contributing.

File: README.md [hunk 2]
```diff
@@ -1,33 +1,54 @@
...
+  * Udp/Tcp/Ipv6 connectivity
+    * Verifies it can talk with a stun server with the given protocol
+  * Data throughput
+    * Establishes a loopback call and tests data channels throughput on the link
+  * Video bandwidth
+    * Establishes a loopback call and tests video performance on the link
+    * Measures rtt on media channels.
+    * Measures bandwidth estimation performance (rampup time, max, average)
+
+## Other tests ##
```
Comment (KaptenJansson) [inline README.md]: Not sure about this one, need to have the same name here and on the actual TestRTC page. Maybe "Manual tests" is fine as it sort of belongs to the same context as the Automatic Tests?

File: README.md (hunk 1 above)
Comment (andresusanopinto) [inline README.md]: Done.

File: README.md (hunk 2 above)
Comment (andresusanopinto) [inline README.md]: Done.

File: README.md [hunk 3]
```diff
@@ -1,33 +1,53 @@
...
+    * Clip detection
+    * Mono mic detection
+* Camera
+  * Check WxH resolution
+    * Checks the camera is able to capture at the requested resolution for 5 seconds
+    * Checks if the frames are frozen or muted/black
+    * Detects how long to start encode frames
+    * Reports encode time and average framerate
+  * Check supported resolutions
+    * Lists other resolutions that appear to be supported
```
Comment (KaptenJansson) [inline README.md]: It actually checks the same resolutions as "Check WxH resolution" does as well. Not sure if that should be changed in the actual test or in this text.

**Comment by KaptenJansson** [comment]
LGTM with nits.

File: README.md (hunk 3 above)
Comment (andresusanopinto) [inline README.md]: Done. Removed *other*

**Pull request merged**